stigs/network/cnp-mysql-stig-v-235146.yaml

```diff
@@ -0,0 +1,22 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
```

Comment:
@tamilmaran-7 since this is a cilium policy it is not required to mention about kubearmor
cc: @raviknox @salman-accuknox @syeddhadi

Reply:
@vishnusomank Okay I will make the changes.